input/size_test.go

```diff
@@ -1,15 +1,41 @@
 package input_test
 
 import (
+	"math/big"
 	"testing"
 
 	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/btcec"
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/txscript"
 	"github.com/btcsuite/btcd/wire"
 	"github.com/btcsuite/btcutil"
 
 	"github.com/lightningnetwork/lnd/input"
+	"github.com/lightningnetwork/lnd/keychain"
+)
+
+const (
+	testCSVDelay = (1 << 31) - 1
+
+	testCLTVExpiry = 500_000_000
```

Comment:
I don't think we can start to use this yet as this literals were introduced in Go 1.13, and our `go.mod` states the lowest version we support atm is Go 1.12.

Reply:
indeed you're correct, fixed